vllm/entrypoints/openai/api_server.py

```diff
@@ -629,6 +629,8 @@ async def completion_stream_generator() -> AsyncGenerator[str, None]:
                         usage=final_usage,
                     )
                     yield f"data: {response_json}\n\n"
+                else:
+                    yield f"data: {response_json}\n\n"
```

Comment:
why not
```suggestion
                yield f"data: {response_json}\n\n"
```

Reply:
@Yard1 Thank you for your suggestion. It has been modified